### PR TITLE
[wip] Supersede work for duplicate records

### PR DESCRIPTION
**Pull request by stepps00**
WIP. This will fix (most of) #1266.

This PR will include the following:
- Superseding one record into the other, bringing any qs, gn, concordances, or alt names over.

This work is only being done on duplicate pairs with one or two point geometries. This work is not being done on duplicate sets of three or more records, or duplicate sets that have two or more geometries that need to be merged together.

To do:
- [x] Tag things as current
- [x] Scrub name fields, saving some names in the variant namespace.

### CODE REVIEW

File: data/101/712/549/101712549.geojson [hunk 1]
```diff
@@ -12,13 +12,17 @@
...
+    "gn:gn_pop":11169,
+    "gn:population":10899,
     "iso:country":"US",
     "lbl:latitude":41.433533,
     "lbl:longitude":-81.75184,
     "mps:latitude":41.433533,
     "mps:longitude":-81.75184,
     "mz:hierarchy_label":1,
-    "mz:is_current":-1,
+    "mz:is_current":1,
```
Comment (nvkelso) [inline data/101/712/549/101712549.geojson]: This is probably too aggressive across the entire PR, suggest leaving `mz:is_current` at -1.

File: data/101/712/549/101712549.geojson [hunk 2]
```diff
@@ -207,13 +232,17 @@
     "wof:concordances":{
         "fct:id":"08d085c2-8f76-11e1-848f-cfd5bf3ef515",
         "fips:code":"3909246",
-        "gn:id":5110302,
+        "gn:id":5148346,
```
Comment (nvkelso) [inline data/101/712/549/101712549.geojson]: Please leave existing concordances alone, and add new ones from the dup record to `wof:concordances_alt` (the opposite of how you've done it here).

File: data/101/712/819/101712819.geojson [hunk 3]
```diff
@@ -16,7 +16,7 @@
     "mps:latitude":40.200754,
     "mps:longitude":-84.800907,
     "mz:hierarchy_label":1,
-    "mz:is_current":-1,
+    "mz:is_current":1,
```
Comment (nvkelso) [inline data/101/712/819/101712819.geojson]: Unsetting the is_current commit should remove this feature from the diff, good!

File: data/101/725/033/101725033.geojson [hunk 4]
```diff
@@ -139,10 +139,10 @@
         "\u0cb2\u0cbe\u0cb0\u0cc6\u0ca1\u0cca"
     ],
     "name:kor_x_preferred":[
-        "\ub77c\ub808\ub3c4"
+        "\ub7ec\ub808\uc774\ub3c4"
```
Comment (nvkelso) [inline data/101/725/033/101725033.geojson]: We should leave the preferred alone instead of swapping these.

Comment (nvkelso) [reply data/101/725/033/101725033.geojson]: (prefer value in the "master" record instead of the random records being superseded into it.)

File: data/101/803/089/101803089.geojson [hunk 5]
```diff
@@ -178,7 +178,7 @@
         }
     ],
     "wof:id":101803089,
-    "wof:lastmodified":1527642767,
+    "wof:lastmodified":1533771393,
```
Comment (nvkelso) [inline data/101/803/089/101803089.geojson]: Why was this modified?

File: data/101/712/549/101712549.geojson (hunk 1 above)
Comment (stepps00) [reply data/101/712/549/101712549.geojson]: 👍

File: data/101/712/549/101712549.geojson (hunk 2 above)
Comment (stepps00) [reply data/101/712/549/101712549.geojson]: Noting that this occurred because I was (incorrectly) preferring the deprecated feature's concordances, instead of the current feature's concordances. Swapped logic in import script, will rerun...

File: data/101/713/017/101713017.geojson [hunk 6]
```diff
@@ -66,6 +69,9 @@
     "name:srp_x_preferred":[
         "\u0410\u043d\u0442\u0438\u043e\u043a"
     ],
+    "name:srp_x_variant":[
+        "\u0410\u043d\u0442\u0438\u043e\u043a "
```
[the remark this answers is not in the document]
Comment (stepps00) [reply data/101/713/017/101713017.geojson]: I'll add logic to catch trailing spaces and remove names when they are exact (minus that trailing space).

Follow-up cases will occur through: https://github.com/whosonfirst-data/whosonfirst-data/issues/1087

**Comment by stepps00** [comment]
Closing and replacing....
